src/components/DateTime/relative/timeToStringRelative.jsx

```diff
@@ -1,11 +1,7 @@
-import { timeToStringCommon } from '../utils/timeToStringCommon.jsx';
+import { getFormattedTimeRelative } from "./getFormattedTimeRelative.jsx";
 
-// Relative time formatting without "ago"
+// Simplifies original timeToStringRelative
 export const timeToStringRelative = (value, dateTimeFormat = "medium", addSuffix = false, locale = undefined) => {
```

Comment:
This is a minor change. Could you please use single quotation marks instead of double ones?

`dateTimeFormat = "medium"` ==> `dateTimeFormat = 'medium'`